code/game/objects/items/beacons.dm

```diff
@@ -0,0 +1,113 @@
+/obj/item/sup_beacon
+	name = "beacon"
+	desc = "N.T. approved supply beacon, but this one shouldn't exist!"
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "supply_beacon"
+	///Has the beacon been used already?
+	var/used
+	///What is the thing this beacon sends to the station?
+	var/obj/beacon_contents = /obj/item/reagent_containers/food/snacks/cookie
+
+/obj/item/sup_beacon/attack_self()
+	if(used)
+		return
+	loc.visible_message("<span class='warning'>\The [src] begins to beep loudly!</span>")
+	used = TRUE
+	addtimer(CALLBACK(src, .proc/launch_payload), 40)
+
+/**
+  * Creates a new beacon_contents object, shoves it in a supply pod, and launches it at the station from nullspace.
+  */
+/obj/item/sup_beacon/proc/launch_payload()
+	var/obj/structure/closet/supplypod/centcompod/toLaunch = new()
+	if(!beacon_contents)
+		return
+	new beacon_contents(toLaunch)
+	new /obj/effect/pod_landingzone(drop_location(), toLaunch)
+	qdel(src)
+
+//******SERVICE EQUIPMENT******
+
+///Roulette wheel
+/obj/item/sup_beacon/roulette
+	name = "roulette wheel beacon"
+	desc = "N.T. approved roulette wheel beacon, toss it down and you will have a complementary roulette wheel delivered to you."
+	beacon_contents = /obj/machinery/roulette
+
+///Beer Dispenser
+/obj/item/sup_beacon/beer_dispenser
+	name = "beer dispenser beacon"
+	desc = "N.T. approved alcohol dispenser beacon, toss it down and you'll have a fully ready bar in no time. Remember to ID everyone!"
+	beacon_contents = /obj/machinery/chem_dispenser/drinks/beer
+
+///Soda Dispenser
+/obj/item/sup_beacon/soda_dispenser
+	name = "soda dispenser beacon"
+	desc = "N.T. approved soda dispenser beacon, toss it down and you'll have a fully ready drink station. Drink cola to feel SUPER!"
+	beacon_contents = /obj/machinery/chem_dispenser/drinks
+
+///Microwave
+/obj/item/sup_beacon/microwave
+	name = "microwave beacon"
+	desc = "N.T. approved microwave beacon, toss it down and you'll have a fully ready kitchen. Okay, well not FULLY ready, but it worked in college, right?"
+	beacon_contents = /obj/machinery/microwave
+
+///Food processor
+/obj/item/sup_beacon/processor
+	name = "food processor beacon"
+	desc = "N.T. approved food processor beacon, toss it down and you'll have a fully ready food prep machine. It slices, it dices, it even makes julianne fries!"
+	beacon_contents = /obj/machinery/processor
+
+/obj/item/sup_beacon/hydroponics
+	name = "hydroponics tray beacon"
+	desc = "N.T. approved hydroponics tray beacon, toss it down and you'll have a fully ready hydroponics tray. Grow's just about anything the stomache desires."
+	beacon_contents = /obj/machinery/hydroponics
+
+//******Medical******
+
+///Statis Bed
+/obj/item/sup_beacon/stasis
+	name = "stasis bed beacon"
+	desc = "N.T. approved stasis bed beacon, toss it down and you'll have a fully ready statis bed. Turn anywhere into an operating theator! Or don't!"
```

Comment:
```suggestion
	desc = "N.T. approved stasis bed beacon, toss it down and you'll have a fully ready stasis bed. Turn anywhere into an operating theator! Or don't!"
```

Reply:
Isn't it theater or theatre? No o I believe

Reply:
yup lol - edited